index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="/css/styles.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Arturo Alvarez</title>
+</head>
+<body>
+	<header class="header">
+		<div class="logo">
+			My Logo
+		</div>
+		<div class="menu">
+			<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6">
+				<path fill-rule="evenodd" d="M3 6.75A.75.75 0 013.75 6h16.5a.75.75 0 010 1.5H3.75A.75.75 0 013 6.75zM3 12a.75.75 0 01.75-.75h16.5a.75.75 0 010 1.5H3.75A.75.75 0 013 12zm0 5.25a.75.75 0 01.75-.75h16.5a.75.75 0 010 1.5H3.75a.75.75 0 01-.75-.75z" clip-rule="evenodd" />
+			</svg>
+		</div>
+	</header>
+	<main>
+		<section class="greet">
+			<div class="title">
+				<p>Welcome my portfolio</p>
+				<h1>Hi! I'm a web developer</h1>
+			</div>
+			<div class="me">
+					<p>
+						My name is Arturo Alvarez and I’m a web developer, I started my career by learning code by myself, and since I began to learn I have worked on many personal projects and also as a freelance for startups here in my country, I have a great enthusiasm for work with great companies worldwide and to learn their professional practices to become a better developer, so I hope that we can work together on many projects.
+					</p>
+			</div>
+			<div class="contact">
+				<p>LET'S CONNECT</p>
+				<div class="social">
+					<img src="/assets/twitter.svg" alt="Twitter logo">
+					<img src="/assets/Linkedin icon.svg" alt="Linkedin logo">
+					<img src="/assets/Vector.svg" alt="Github logo">
+				</div>
```

Comment:
Please use `ul` and `li` tags to display the icons and also each icon should have an `a` tag so it can be clicked on to your social media handles.